<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open-Graphtheory.org</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuGenerate">
     <property name="title">
      <string>Generate</string>
     </property>
     <addaction name="actionGenerate"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="menuGenerate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
    <addaction name="actionSpring_Embed"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionForum"/>
    <addaction name="actionBugtracker"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithms">
    <property name="title">
     <string>Algorithms</string>
    </property>
    <addaction name="actionClique"/>
    <addaction name="actionIndependent_Set"/>
    <addaction name="actionDominating_Set"/>
    <addaction name="actionMaximum_Matching"/>
    <addaction name="actionColoring"/>
    <addaction name="actionPath"/>
    <addaction name="actionDisjoint_Paths"/>
    <addaction name="actionMinimum_Cut"/>
    <addaction name="actionMinimum_Separator"/>
    <addaction name="actionVertex_Cover"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuLogic">
    <property name="title">
     <string>Logic</string>
    </property>
    <addaction name="actionModal_Logic"/>
    <addaction name="actionComputation_Tree_Logic"/>
    <addaction name="actionFirst_Order_Predicate_Logic"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAlgorithms"/>
   <addaction name="menuLogic"/>
   <addaction name="menuLayout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSpring_Embed">
   <property name="text">
    <string>Spring-Embed</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
  <action name="actionBugtracker">
   <property name="text">
    <string>Bugtracker</string>
   </property>
  </action>
  <action name="actionClique">
   <property name="text">
    <string>Clique</string>
   </property>
  </action>
  <action name="actionIndependent_Set">
   <property name="text">
    <string>Independent Set</string>
   </property>
  </action>
  <action name="actionDominating_Set">
   <property name="text">
    <string>Dominating Set</string>
   </property>
  </action>
  <action name="actionColoring">
   <property name="text">
    <string>Coloring</string>
   </property>
  </action>
  <action name="actionPath">
   <property name="text">
    <string>Path</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionModal_Logic">
   <property name="text">
    <string>Modal-Logic</string>
   </property>
  </action>
  <action name="actionComputation_Tree_Logic">
   <property name="text">
    <string>Computation-Tree Logic</string>
   </property>
  </action>
  <action name="actionFirst_Order_Predicate_Logic">
   <property name="text">
    <string>First-Order Predicate Logic</string>
   </property>
  </action>
  <action name="actionMinimum_Cut">
   <property name="text">
    <string>Minimum Cut</string>
   </property>
  </action>
  <action name="actionMaximum_Matching">
   <property name="text">
    <string>Maximum Matching</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionVertex_Cover">
   <property name="text">
    <string>Vertex Cover</string>
   </property>
  </action>
  <action name="actionMinimum_Separator">
   <property name="text">
    <string>Minimum Separator</string>
   </property>
  </action>
  <action name="actionDisjoint_Paths">
   <property name="text">
    <string>Disjoint Paths</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
